lib/ansible/plugins/doc_fragments/url_redirect.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2018, John Barker <gundalow@redhat.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import annotations
+
+
+class ModuleDocFragment(object):
+
+    # Standard files documentation fragment
+    DOCUMENTATION = r'''
+options:
+  follow_redirects:
+    description:
+      - Whether or not the URI module should follow redirects. V(all) will follow all redirects.
```

Comment:
One problem is that this docs fragment now explicitly mentions the `uri` module. I would reformulate this to
```suggestion
      - Whether or not redirects should be followed. V(all) will follow all redirects.
```